stackslib/src/config/mod.rs

```diff
@@ -2670,6 +2676,18 @@ impl MinerConfigFile {
             activated_vrf_key_path: self.activated_vrf_key_path.clone(),
             fast_rampup: self.fast_rampup.unwrap_or(miner_default_config.fast_rampup),
             underperform_stop_threshold: self.underperform_stop_threshold,
+            mempool_walk_strategy: {
+                if let Some(mempool_walk_strategy) = &self.mempool_walk_strategy {
+                    match str::parse(&mempool_walk_strategy) {
+                        Ok(strategy) => strategy,
+                        Err(e) => {
+                            panic!("could not parse '{mempool_walk_strategy}': {e}");
+                        },
+                    }
+                } else {
+                    MemPoolWalkStrategy::GlobalFeeRate
+                }
+            },
```

Comment:
A more concise way to write this:

```suggestion
            mempool_walk_strategy: self.mempool_walk_strategy
                .map(|s| str::parse(&s).unwrap_or_else(|e| panic!("Could not parse '{s}': {e}")))
                .unwrap_or(MemPoolWalkStrategy::GlobalFeeRate),
```